### PR TITLE
First Timers Bot App 

### PR DESCRIPTION
**Pull request by Techforchange**
Please let me know if there are any changes that you would need from me. Thanks!

### CODE REVIEW

File: _apps/first-timers-bot.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
+---
+title: First Timers Bot
+description: A friendly bot that helps with onboarding new Open Source Contributors
+slug: first-timers-bot
+screenshots:
+- https://raw.githubusercontent.com/hoodiehq/first-timers-bot/master/assets/Issue-Done.png
+author: agonzalez0515 ArlenePerez
```
Comment (hiimbex) [inline _apps/first-timers-bot.md]: Based on the changes in https://github.com/probot/probot.github.io/pull/46 this should be able to become `authors: [ githubUserName, githubUserName ]`

**Comment by arlene-perez** [comment]
@bkeepers Good catch, going to convert them to HTML tables. Thanks!

**Comment by Techforchange** [comment]
@bkeepers Do you mind seeing if the website renders it correctly now that I have updated the code with HTML tables? Thank you!

File: _apps/first-timers-bot.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
+---
+title: First Timers Bot
+description: A friendly bot that helps with onboarding new Open Source Contributors
+slug: first-timers-bot
+screenshots:
+- https://raw.githubusercontent.com/hoodiehq/first-timers-bot/master/assets/Issue-Done.png
+authors: [ agonzalez0515, ArlenePerez ]
```
Comment (bkeepers) [inline _apps/first-timers-bot.md]: Should this be @techforchange instead of @ArlenePerez (looks like an organization)?

Comment (hiimbex) [reply _apps/first-timers-bot.md]: @bkeepers what do you think about having a display author name and a github username field for links? That way anyone who has a github username that isn't connected with their name, but wants credit with their real name can have the best of both worlds? (ie displaying their name, but linking to their github)

**Pull request merged**